File: java-suite/src/main/java/de/fraunhofer/iem/classesAndObjects/classes/abstractClass/Horse.java
package de.fraunhofer.iem.classesAndObjects.classes.abstractClass;

/**
 * Class that extends Animal
 *
 * @author Ranjith Krishnamurthy
 */
public class Horse extends Animal {
    public Horse(String DNA) {
        super(DNA);
    }

    @Override
    public void makeNoise() {
        System.out.println("Neigh");
    }
}

File: kotlin-suite/src/main/kotlin/de/fraunhofer/iem/classesAndObjects/classes/abstractClass/Horse.kt
package de.fraunhofer.iem.classesAndObjects.classes.abstractClass

/**
 * Class that extends Animal
 *
 * @author Ranjith Krishnamurthy
 */
class Horse(DNA: String) : Animal(DNA) {
    override fun makeNoise() {
        println("Neigh")
    }
}